vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -69,7 +70,15 @@ def pre_build(self):
         for order, page in enumerate(self.pages):
             path = page.path if order else "/"
             dash.register_page(
-                module=page.id, name=page.title, path=path, order=order, layout=partial(self._make_page_layout, page)
+                module=page.id,
+                name=page.title,
+                path=path,
+                order=order,
+                layout=partial(self._make_page_layout, page),
+                description=page.description,
+                image=self._infer_image("app")
+                or self._infer_image("logo")
+                or get_relative_path(f"/{STATIC_URL_PREFIX}/images/app.svg"),
```

Comment:
Need to check if this works for the meta tags --- the link looks strange with two slashes after assets - but it works for serving up the 404 image.

![image](https://github.com/mckinsey/vizro/assets/72614349/2aff1e5d-bde4-4d99-8967-88d92204482f)

Reply:
Do you want me to take this out?  Just as long as there isn't an `app.<ext>` in the Vizro assets folder it won't add an image.

Reply:
Yeah, let's take this out for now please, and I will follow up with Joe in the new year.

After you've resolved the conversation (necessary to merge), please can you unresolve it so it's easier to find again when we refer back to it in future? 🙏